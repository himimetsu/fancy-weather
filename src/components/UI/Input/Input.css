.Input {
  width: 280px;
  padding-left: 10px;
  box-sizing: border-box;
  border: 2px solid #ccc;
  border-right: none;
  border-radius: 5px 0 0 5px;
  background: transparent;
  color: white;
}