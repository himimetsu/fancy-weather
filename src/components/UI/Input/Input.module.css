.Input {
  width: 270px;
  padding-left: 10px;
  box-sizing: border-box;
  border: 2px solid #ccc;
  border-right: none;
  border-radius: 5px 0 0 5px;
  background: transparent;
  color: white;
}