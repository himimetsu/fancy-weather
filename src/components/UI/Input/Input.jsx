import React from 'react';
import './Input.css';

const Input = props => {
  return (
    <input {...props} className='search-input' />
  )
};

export default Input;
