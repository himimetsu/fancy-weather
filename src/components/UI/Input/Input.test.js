import React from 'react';
import ReactDOM from 'react-dom';
import Input from './Input';
import { render, cleanup } from '@testing-library/react';

afterEach(cleanup);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Input/>, div);
});

it('mathes snapshot', () => {
  const { asFragment } = render(<Input/>);
  expect(asFragment()).toMatchSnapshot();
});