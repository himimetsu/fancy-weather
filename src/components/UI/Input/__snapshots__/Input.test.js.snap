// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mathes snapshot 1`] = `
<DocumentFragment>
  <input
    class="search-input"
  />
</DocumentFragment>
`;
